nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessAnalysis.java

```diff
@@ -165,7 +165,7 @@ private Set<Element> getNonnullReceiverFields(NullnessStore nullnessResult) {
     Set<AccessPath> nonnullAccessPaths = nullnessResult.getAccessPathsWithValue(Nullness.NONNULL);
     Set<Element> result = new LinkedHashSet<>();
     for (AccessPath ap : nonnullAccessPaths) {
-      if (ap.getRoot().isReceiver()) {
+      if (ap.getRoot() == null) {
```

Comment:
A small downside of this refactoring is that this is slightly less self-documenting. Maybe worth adding a one-line comment here noting that this is the case where the root is the method's receiver?

Reply:
I can add a method `AccessPath.rootIsReceiver()` maybe, so it's still self-documenting?

Reply:
Makes sense to me :) Wouldn't help in the `!= null` check without sophisticated ensures/requires/contract logic we can't handle yet, but that case is already somewhat self-documenting :)